Doc/library/enum.rst

```diff
@@ -117,6 +117,19 @@ Module Contents
 
       Enum class decorator that ensures only one name is bound to any one value.
 
+   :func:`simple_enum`
```

Comment:
I suggest we not make this a public API for at least the first release we see if we want to live with this approach.  So I wouldn't document it yet and would add an _ prefix to both names.  That way if we wind up doing something else later it is easy to undo without a drawn out deprecation.

Reply:
Sounds good.